agent/src/agent/cli/run_test_pipeline.py

```diff
@@ -0,0 +1,103 @@
+import json
+import os
+import time
+import click
+import sdc_client
+
+from jsonschema.exceptions import ValidationError
+from agent import cli, destination, pipeline, source
+from agent.modules import constants
+from agent.modules.logger import get_logger
+
+logger = get_logger(__name__, stdout=True)
+
+
+@click.command(name='run-test-pipeline')
+@click.option('--no-delete', is_flag=True, help='Delete test Source and Pipeline.')
+def run_test_pipeline(no_delete):
+    """
+    Creates temporary source and pipeline to check the destination is accessible.
+    """
+    try:
+        cli.pipeline.check_prerequisites()
+        _add_source()
+        _add_pipeline()
+        _run_pipeline()
+    except click.ClickException as e:
+        click.secho(f'Test run failed: {e}', fg='red')
+        exit(1)
+    else:
+        click.secho('Test run completed', fg='green')
+    finally:
+        if not no_delete:
+            perform_cleanup()
+
+
+def _add_source():
+    """
+    Creates temporary source for run-test-pipeline command
+    """
+    source_file = os.path.join(constants.LOCAL_RUN_TESTPIPELINE_DIR, "sources", "test_source.json")
+    try:
+        with open(source_file, "r") as file:
+            source.json_builder.create_from_file(file)
+    except source.SourceException as e:
+        _check_exist(e)
```

Comment:
checking text in errors can be problematic because it may change and we won't know about it. Why don't just show the error as is?

Reply:
It was just for testing. Already fixed